projects/packages/publicize/src/class-connections.php

```diff
@@ -189,7 +189,7 @@ public static function fetch_and_cache_connections() {
 	public static function fetch_site_connections() {
 		$proxy = new Proxy_Requests( 'publicize/connections' );
 
-		$request = new WP_REST_Request( 'GET', '/wpcom/v2/publicize/connections' );
```

Comment:
While testing the changes here, I found out that this part is unnecessary and thus cleaned it up.